lib/knapsack_pro/runners/base_runner.rb

```diff
@@ -35,6 +35,10 @@ def test_files_to_execute_exist?
 
       attr_reader :allocator_builder,
         :allocator
+
+      def self.child_status
+        $?
```

Comment:
`$?` has been extracted to a method `child_status` because it's not possible anymore to mock `$?` so let's hide it inside of a method that we can mock.